src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/FirstEditNag.css

```diff
@@ -0,0 +1,4 @@
+.first-edit-nag {
+  background: var(--theme-popup-color);
+  color: var(--theme-popup-background);
```

Comment:
I think this'll match [LineNumberTooltip](https://github.com/RecordReplay/devtools/blob/81d9a94fbbcdeeafeb1b13b4a895f5d8c5072641/src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.css#L9-L10).

@jonbell-lot23 do you get a little bit of an optical illusion where it looks like there's not border here or is that just me? Maybe I can try to make the top border match the nag background or something.

<img width="496" alt="Screen Shot 2021-10-20 at 3 54 27 PM" src="https://user-images.githubusercontent.com/478109/138162655-0933c418-d746-4961-9679-aec9c042d899.png">

Reply:
@colbyr I wonder if we should go in the other direction and make both nags blue which I think might be more playful and more clearly a call out.

It could also be fun to do other hover effects, like hovering in the panel makes the stars glisten yellow, even sparkle... 

This is probably a separate issue, but we should also look at the hover area for the expression because even though the entire box is clickable, only a portion changes on hover and we can make the hover effect more obvious. All of this is coming from the place that half of the users who add a breakpoint actually edit it. We should get that # as high as possible.